tests/test_integration.py

```diff
@@ -783,7 +783,7 @@ def test_universe_data_formatter(self):
         with self.assertWarns(LineExpansionWarning):
             output = problem.cells._universe.format_for_mcnp_input((6, 2, 0))
         print(output)
-        self.assertIn("u 350 2J -1 J 350", output)
+        self.assertIn("u 350 2J -1 J 350 ", output)
```

Comment:
So that I follow: the addition of the space is testing the `never_pad` behavior?

Reply:
The `never_pad` behavior slightly changed, and in this case a trailing space is added. I figure the users won't care, and it would be really complicated to suppress it in this case. So I just said this is the new gold for the test.